fastlane/swift/SocketResponse.swift

```diff
@@ -35,12 +35,16 @@ struct SocketResponse {
                 return
 
             } else if status == "failure" {
-                guard let failureInformation = statusDictionary["failure_information"] as? [String] else {
+                guard
+                  let failureInformation = statusDictionary["failure_information"] as? [String],
+                  let failureClass = statusDictionary["failure_class"] as? String?,
+                  let failureMessage = statusDictionary["failure_message"] as? String?
```

Comment:
`failureClass` and `failureMessage` are optional in the num. Do these really need to be in the `guard`? 🤔

Reply:
Is the most straightforward way to assign them into a optional variable to use them afterwards instead of searching them into the dictionary in the following lines and make the cast there, just a way of making the information more readable, but the other approach is as valid as the current one. Just a matter of readability for short. But I can change it if you want 😄